### PR TITLE
grc: 1.9 -> 1.10.1

### PR DESCRIPTION
**Pull request by peterhoeg**
###### Motivation for this change

Additionally, grc wasn't actually working unless you copied out the config files to your home directory because it couldn't find the default config in /etc/grc.conf

So now we patch the program to look in the right places.

Any comments @lovek323 or @AndersonTorres ?

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/misc/grc/default.nix
Comment (matthiasbeyer) [inline pkgs/tools/misc/grc/default.nix]: Why not fetching from here anymore? Maybe a comment in the commit?

**Review by matthiasbeyer — CHANGES_REQUESTED** [review]

File: pkgs/tools/misc/grc/default.nix
Comment (peterhoeg) [inline pkgs/tools/misc/grc/default.nix]: Upstream moved. I've added a note in the commit.

**Review by matthiasbeyer — APPROVED** [review]
I approve, though I did not test-compile this. But what I suggested above was resolved, so ACK from me.

**Comment by peterhoeg** [comment]
As both @lovek323 and @AndersonTorres are listed as maintainers, I would really like some feedback before I merge this.

**Comment by grahamc** [comment]
Sounds like they're not coming, Peter. Want to go ahead and merge?

**Pull request merged**
